pkg/kclient/deployments.go

```diff
@@ -115,6 +116,40 @@ func (c *Client) ListDeployments(selector string) (*appsv1.DeploymentList, error
 	})
 }
 
+func (c *Client) WaitForPodNotRunning(name string) error {
+	watch, err := c.KubeClient.CoreV1().Pods(c.Namespace).Watch(context.TODO(), metav1.ListOptions{FieldSelector: "metadata.name=" + name})
+	if err != nil {
+		return err
+	}
+	defer watch.Stop()
+
+	if _, err = c.KubeClient.CoreV1().Pods(c.Namespace).Get(context.TODO(), name, metav1.GetOptions{}); kerrors.IsNotFound(err) {
+		fmt.Printf("\npod not found\n")
```

Comment:
I think we don't need this print statement

Reply:
sure, some debug remains